tests/monitor.unit.spec.ts

```diff
@@ -15,17 +15,26 @@ const temperature = 1;
 
 const createDelayedResponse =
   (result: any): ((...args: any[]) => Promise<any>) =>
-  () =>
-    new Promise((resolve) => setTimeout(() => resolve(result), 1));
+    () =>
+      new Promise((resolve) => setTimeout(() => resolve(result), 1));
 
 describe('monitorOpenAI', () => {
+  //@ts-ignore
   let openai: OpenAIApi;
 
   beforeEach(() => {
+
+    //@ts-ignore
```

Comment:
The clean way to do this is to use the `__mocks__` folder.
Look at how I mocked the `@newrelic/telemetry-sdk`.